.travis.yml

```diff
@@ -2,14 +2,22 @@ language: node_js
 node_js:
   - 'node'
 env:
-  - EXTENSION_ID='extension id for publising to CWS'
+  - EXTENSION_ID='extension id for publising to CWS' RELEASE_ON_TAGS='false'
```

Comment:
Place `RELEASE_ON_TAGS` on new line.

Reply:
I can't do this because this will trigger Travis CI build matrix.